src/input-helper.ts

```diff
@@ -106,6 +106,12 @@ export function getInputs(): ISourceSettings {
   // Auth token
   result.authToken = core.getInput('token')
 
+  // SSH
+  result.sshKey = core.getInput('ssh-key')
+  result.sshKnownHosts = core.getInput('ssh-known-hosts')
+  result.sshStrict =
```

Comment:
I think we should default to True if there is no input to match the default value, I also feel safer defaulting to the more secure option